ntoskrnl/ps/process.c

```diff
@@ -169,6 +169,52 @@ PsGetNextProcess(IN PEPROCESS OldProcess)
     return FoundProcess;
 }
 
+PEPROCESS
+NTAPI
+PsGetPreviousProcess(IN PEPROCESS OldProcess)
+{
+    PLIST_ENTRY Entry;
+    PEPROCESS FoundProcess = NULL;
+    PAGED_CODE();
```

Comment:
You can put a newline before the PAGED_CODE() and stuff, so that we can see very quickly where the variables list stop and where these stuff start.

Reply:
This is copy-paste from `PsGetNextProcess`.